spec/features/user_signs_up_spec.rb

```diff
@@ -7,17 +7,20 @@
 } do
 
   # Acceptance Criteria:
-  # * I must specify a valid email address,
+  # * I must specify a valid email address, username,
   #   password, and password confirmation
+  # * Email, username, and password must be unique
+  # * I can optionally provide an avatar
   # * If I don't specify the required information, I am presented with
   #   an error message
 
-  scenario 'provide valid registration information' do
+  scenario 'provide valid registration information - no avatar' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.